src/app/(admin)/bundles/components/Bundle.tsx

```diff
@@ -26,6 +26,13 @@ import { useUiStore } from "src/app/stores/UiStore"
 
 import { BundleExportUpdate } from "src/app/jobs/BundleExportUpdate"
 
+const generatePublicBundleLink = (bundle: any) => {
+  const importMetadata = bundle.importMetadata as { grade?: string, unit?: string };
+  const grade = importMetadata.grade || 'unknown-grade';
```

Comment:
I don't think that it makes sense to generate a link if we don't know the grade/unit. Maybe we should throw?